tools/functional-tester/etcd-tester/tester.go

```diff
@@ -104,6 +104,11 @@ func (tt *tester) doRound(round int) (bool, error) {
 			plog.Printf("%s recovery error: %v", tt.logPrefix(), err)
 			return false, nil
 		}
+		plog.Printf("%s wait until cluster is fully alive", tt.logPrefix())
+		if err := tt.cluster.WaitHealth(); err != nil {
```

Comment:
before i remove the loop. I wonder is the sole purpose of the loop to make sure that cluster is in full health while checking the revision. Could it is also be used as a retry strategy in case of failure?